src/app/zap-templates/templates/app/cluster-objects-src.zapt

```diff
@@ -173,5 +173,68 @@ bool CommandNeedsTimedInvoke(ClusterId aCluster, CommandId aCommand)
     return false;
 }
 
+// TODO(#20811): Actually generate the following based on ZAP metadata
+// See https://github.com/project-chip/zap/issues/609
+bool CommandIsFabricScoped(ClusterId aCluster, CommandId aCommand)
```

Comment:
This does not seem to depend on template inputs. It should be in a separate header/source

Reply:
This will be generated this week once ZAP has the correct support. See https://github.com/project-chip/zap/issues/609 and https://github.com/project-chip/zap/pull/611